index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>JSON Tester</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+
+<body onload="renderHello()">
+  <script src="https://unpkg.com/mustache@latest"></script>
+  <script src="render.js"></script>
+  <script src="resume.js"></script>
+
+  <div id="target">Loading...</div>
+  <script id="template" type="x-tmpl-mustache">
+    <div class="header">
+      <h1> {{ resume.basics.name }}</h1>
+      <div>
+        <b>Email:</b> {{ resume.basics.email }}
+        <b>Mobile:</b> {{ resume.basics.phone }}
```

Comment:
I noticed there is some inconsistency with the spacing of the templates variables. Some are formatted with spacing and others aren't. Just something to look out for!

Reply:
My auto formatted did not like anything inside the script block. Most of the spacing issues are due to not inspecting and just pushing to get someone up for review. Good eye with spotting it.